noble/chain.json

```diff
@@ -166,7 +166,7 @@
   },
   "logo_URIs": {
     "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/stake.png",
-    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/Noble-Logo.svg"
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/stake.svg"
```

Comment:
Can you please update the same svg path in the `images` object as well?

Reply:
Done! @JeremyParish69